src/main/java/org/myrobotlab/service/InMoov2.java

```diff
@@ -640,11 +629,16 @@ public InMoov2Arm getLeftArm() {
 	public InMoov2Hand getLeftHand() {
 		return leftHand;
 	}
+	
+  @Override
+  public String getLanguage() {
+    return Runtime.getInstance().getLanguage();
+  }
 
-	@Override
-	public Locale getLocale() {
-		return locale;
-	}
+  @Override
+  public Locale getLocale() {
+    return Runtime.getInstance().getLocale();

```

Comment:
This should be the locale of the InMoov2 service.. not Runtime.

Reply:
wow .. i can't believe I left this in .. thanks - again .. nice catch.. removed